tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -7384,7 +7384,7 @@ public function testPostLinkWithConfirm()
             ],
             'input' => ['type' => 'hidden', 'name' => '_method', 'value' => 'POST'],
             '/form',
-            'a' => ['href' => '#', 'onclick' => "preg:/if \(confirm\(&quot;&#039;Confirm&#039;\\nthis \\\\&quot;deletion\\\\&quot;\?&quot;\)\) \{ document\.post_\w+\.submit\(\); \} event\.returnValue = false; return false;/"],
+            'a' => ['href' => '#', 'onclick' => "preg:/if \(confirm\(&quot;&#039;Confirm&#039;\\nthis \\&quot;deletion\\&quot;\?&quot;\)\) \{ document\.post_\w+\.submit\(\); \} event\.returnValue = false; return false;/"],
```

Comment:
So I edited the FormHelperTest now? So that the Test is working. I don't really know if the test qorked before, but I don't think so because now this test is working and with 4 backward slashes it's not.